src/components/DropdownForm/DropdownForm.tsx

```diff
@@ -0,0 +1,58 @@
+import React, {
+  useCallback, useEffect, useRef, useState,
+} from 'react';
+import debounce from 'lodash.debounce';
+import { Person } from '../../types/Person';
+
+type Props = {
+  querry: string;
+  onQuerry: (value: string) => void;
+  onAppliedQuerry?: (value: string) => void;
+  selectedPerson: Person | null,
+  onDropdownFocus?: (value: boolean) => void;
+  debounceTimer: number;
+};
+
+export const DropdownForm: React.FC<Props> = ({
+  querry,
+  onQuerry,
+  onAppliedQuerry = () => { },
+  selectedPerson,
+  onDropdownFocus = () => { },
+  debounceTimer,
+}) => {
+  const [defoultImputFocus, setDefoultImputFocus] = useState(false);
```

Comment:
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig
```suggestion
  const [isInputInFocus, setIsInputInFocus] = useState(false);
```